Packs/HealthCheck/Dashboards/dashboard-c7f84a7b-341b-4e9d-8911-1df1369d1d69-XSOAR_Status_Review_Dashboard_v6.json

```diff
@@ -0,0 +1,2626 @@
+                    "period": {
+                        "by": "",
+                        "byFrom": "",
+                        "byTo": "",
+                        "field": "",
+                        "fromValue": null,
+                        "toValue": null
+                    },
+                    "toDate": "0001-01-01T00:00:00Z"
+                },
+                "definitionId": "",
+                "fromServerVersion": "",
+                "id": "6b7f41d6-db64-4f17-8ef4-df575cf45f2b",
+                "indexName": "testprefix-dmst-common-configuration",
+                "isPredefined": false,
+                "itemVersion": "",
+                "modified": "2024-04-24T10:03:07.953047425Z",
+                "name": "Playbooks Health Status",
+                "packID": "",
+                "packName": "",
+                "params": {
+                    "tableColumns": []
+                },
+                "prevName": "Playbooks Health Status",
+                "primaryTerm": 15,
+                "propagationLabels": [
+                    "all"
+                ],
+                "query": "getPlaybooksHealthStatus",
+                "sequenceNumber": 483646899,
+                "shouldCommit": false,
+                "sizeInBytes": 633,
+                "toServerVersion": "",
+                "vcShouldIgnore": false,
+                "vcShouldKeepItemLegacyProdMachine": false,
+                "version": 1,
+                "widgetType": "table"
+            },
+            "x": 0,
+            "y": 17
+        },
+        {
+            "forceRange": true,
+            "h": 3,
+            "i": "36c6b330-0222-11ef-9e9d-3d80c630135f",
+            "id": "36c6b330-0222-11ef-9e9d-3d80c630135f",
+            "reflectDimensions": true,
+            "w": 4,
+            "widget": {
+                "Cache": null,
+                "cacheVersn": 0,
+                "category": "",
+                "commitMessage": "",
+                "created": "2024-04-24T10:05:43.008856991Z",
+                "dataType": "scripts",
+                "dateRange": {
+                    "fromDate": "0001-01-01T00:00:00Z",
+                    "fromDateLicense": "0001-01-01T00:00:00Z",
+                    "period": {
+                        "by": "",
+                        "byFrom": "days",
+                        "byTo": "",
+                        "field": "",
+                        "fromValue": null,
+                        "toValue": null
+                    },
+                    "toDate": "0001-01-01T00:00:00Z"
+                },
+                "definitionId": "",
+                "fromServerVersion": "",
+                "id": "1f6ce26d-914c-4769-804b-6b04163887da",
+                "indexName": "testprefix-dmst-common-configuration",
+                "isPredefined": false,
+                "itemVersion": "",
+                "modified": "2024-04-24T10:05:43.013939789Z",
+                "name": "Big Incidents and Indicators",
```

Comment:
Getting this error on the V6 - 
![image](https://github.com/demisto/content/assets/88268646/60a6feb2-ef78-4454-a1ad-04a73b0c57d5)

```Script failed to run: 
Error: [Traceback (most recent call last):
   
   File "<string>", line 6, in <module>
   File "<string>", line 96, in executeCommand
   File "<string>", line 189, in __do
 ValueError: Unsupported Command : core-api-get , verify you have proper integration enabled to support it (23)

] (2604) (2603)

Reply:
Having CORE REST API integration enabled is the pre requisite before using this dashboard

Reply:
@ssokolovich Also I couldn't get your comment - "Some of the widgets are trying to fetch data using both commands - for x6 and x8."

Reply:
My bad - I forgot that the new API integration is named core (got confused with XSAIM and native commands in xsoar v8). Ignore this comment will review it again with the integration enabled.

Side note - please mention that on the Dashboard's description as well 😄

Reply:
@ssokolovich NO worries :) 
1. Added the description to both the dashboards
2. Updated the toVersion to 6.99.0
3. Removed the percentage signs from the release notes
4. Removed the multiple demisto.args() from the scripts
5. Updated the v8 dashboard fromVerison with 8.0.0